specification/communication/data-plane/Chat/preview/2022-11-15-preview8/communicationserviceschat.json

```diff
@@ -0,0 +1,1977 @@
+            "example": "2020-10-30T10:50:50Z"
+          },
+          "lastMessageReceivedOn": {
+            "format": "date-time",
+            "description": "The timestamp when the last message arrived at the server. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+            "type": "string",
+            "readOnly": true,
+            "example": "2020-10-30T10:50:50Z"
+          }
+        }
+      },
+      "ChatThreadsItemCollection": {
+        "description": "Collection of chat threads.",
+        "required": [
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "value": {
+            "description": "Collection of chat threads.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/ChatThreadItem"
+            }
+          },
+          "nextLink": {
+            "description": "If there are more chat threads that can be retrieved, the next link will be populated.",
+            "type": "string",
+            "readOnly": true
+          }
+        }
+      },
+      "UpdateChatThreadRequest": {
+        "description": "Request payload for updating a chat thread.",
+        "type": "object",
+        "properties": {
+          "topic": {
+            "description": "Chat thread topic.",
+            "type": "string",
+            "example": "Lunch Thread"
+          }
+        }
+      },
+      "SendTypingNotificationRequest": {
+        "description": "Request payload for typing notifications.",
+        "type": "object",
+        "properties": {
+          "senderDisplayName": {
+            "description": "The display name of the typing notification sender. This property is used to populate sender name for push notifications.",
+            "type": "string",
+            "example": "Bob Admin"
+          }
+        }
+      }
+    },
+    "parameters": {
+      "ApiVersionParameter": {
+        "in": "query",
+        "name": "api-version",
+        "description": "Version of API to invoke.",
+        "required": true,
+        "type": "string",
+        "x-ms-parameter-location": "method"
+      },
+      "Endpoint": {
+        "in": "path",
+        "name": "endpoint",
+        "description": "The endpoint of the Azure Communication resource.",
+        "required": true,
+        "type": "string",
+        "x-ms-skip-url-encoding": true,
+        "x-ms-parameter-location": "client"
+      }
+    },
+    "securityDefinitions": {
+      "Authorization": {
+        "type": "apiKey",
+        "name": "Authorization",
+        "in": "header",
+        "description": "An ACS (Azure Communication Services) user access token."
+      }
+    },
+    "security": [
+      {
+        "Authorization": []
+      }
+    ],
+    "x-ms-parameterized-host": {
+      "hostTemplate": "{endpoint}",
+      "useSchemePrefix": false,
+      "parameters": [
+        {
+          "$ref": "#/parameters/Endpoint"
+        }
+      ]
+    }
+  }
```

Comment:
This json is invalid and is missing a closing curly brace at the end. Do you have swagger UI installed in your VS Code?

Reply:
It seems that there are other things too that are malformed, Swagger UI can't render the full document